docker/Dockerfile.ubuntu

```diff
@@ -17,18 +17,12 @@ WORKDIR /root/bcc
 RUN /usr/lib/pbuilder/pbuilder-satisfydepends && \
     ./scripts/build-deb.sh ${BUILD_TYPE}
 
-FROM ubuntu:${OS_TAG}
+FROM debian:bookworm
 
 COPY --from=builder /root/bcc/*.deb /root/bcc/
 
 RUN \
   apt-get update -y && \
-  DEBIAN_FRONTEND=noninteractive apt-get install -y python python3 python3-pip binutils libelf1 kmod  && \
-  if [ ${OS_TAG} = "18.04" ];then \
-    apt-get -y install python-pip && \
-    pip install dnslib cachetools ; \
-  fi ; \
-  pip3 install dnslib cachetools  && \
-  pip3 install -U pip setuptools wheel && \
-  apt-get -y remove python3-pip python3-wheel python3-setuptools && \
-  dpkg -i /root/bcc/*.deb
+  DEBIAN_FRONTEND=noninteractive apt-get install -y binutils libelf1 libtinfo5 python3 xz-utils && \
```

Comment:
kmod is needed. Otherwise I get:

```bash
/usr/share/bcc/tools/execsnoop 
sh: 1: modprobe: not found
Unable to find kernel headers. Try rebuilding kernel with CONFIG_IKHEADERS=m (module) or installing the kernel development package for your running kernel version.
```

Reply:
> Indeed, this is the host responsibilities to provide them:

Where is this specified? kmod is present on the upstream repository https://github.com/iovisor/bcc/blob/963e4a6489f6e858aad9d9e8fcde823bf0a649b0/docker/Dockerfile.ubuntu#LL26C97-L26C101

> The container will not be able to know the host kernel 

I tried installing kmod and running the tool from the container and it works fine

> and there will be problems if we try to modprobe from the container.

What kind of problems could happen?